hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/fs/contract/AbstractContractAppendTest.java

```diff
@@ -149,4 +150,31 @@ public void testRenameFileBeingAppended() throws Throwable {
                                                  dataset.length);
     ContractTestUtils.compareByteArrays(dataset, bytes, dataset.length);
   }
+
```

Comment:
HADOOP-15744 needs to be applied to this branch; else HDFS tests will break.

Reply:
Yes I have applied to it.